src/components/MatPass.tsx

```
@@ -0,0 +1,478 @@
+  }
+
+  if (bundleError) {
+    return (
+      <div className="flex-1 p-6">
+        <div className="p-6 text-center">
+          <p className="text-red-600 mb-4">{bundleError.message}</p>
+          <button
+            onClick={() => window.location.reload()}
+            className="px-4 py-2 bg-blue-500 text-white rounded-lg hover:bg-blue-600 transition-colors"
+          >
+            {language === "en" ? "Try Again" : "Intentar de nuevo"}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)